src/implementation/MassaWalletProviders.ts

```diff
@@ -0,0 +1,120 @@
+import { uid } from 'uid';
+import { ICustomEventMessageResponse } from '../interfaces/ICustomEventMessageResponse';
+import { ICustomEventMessageRequest } from '../interfaces/ICustomEventMessageRequest';
+import { AvailableCommands } from './Commands';
+import { IRegisterEvent } from '../interfaces';
+
+const MASSA_WINDOW_OBJECT = 'massaWalletProvider';
+
+type CallbackFunctionVariadicAnyReturn = (
+  error: Error | null,
+  result: any,
+) => any;
+
+// =========================================================
+
+export class MassaWalletProviders {
```

Comment:
I think this class should be named Connector

Reply:
renamed